src/client/websocket/WebSocketManager.js

```diff
@@ -220,7 +220,6 @@ class WebSocketManager extends EventEmitter {
           this.debug(`Session id is present, attempting an immediate reconnect...`, shard);
           this.reconnect();
         } else {
-          shard.destroy({ reset: true, emit: false, log: false });
           this.reconnect();
```

Comment:
Might as well simplify this and always call this.reconnect outside the if/else (but keep the if case to log that message maybe? if its even useful...

Reply:
Yeah, the debug log is helpful to know if the `{ reset: true }` did its job.